action.yml

```diff
@@ -27,13 +27,13 @@ inputs:
     required: false
     default: 'builtin://Recommended .NET Core Rules'
   resource:
-    description: 'Solution-relative path to a project in a solution, a directory of files in a project, or a file.'
+    description: 'Solution-relative path to a project in a solution, a directory of files in a project, or a file. Use semicolon-separated list to specify many.'
```

Comment:
Use a semicolon-separated list to specify multiple paths.

Reply:
Done.